MTL-1748 Update PoR to Align with NCN Behavior (#1665)
Cray-HPE/docs-csm: background/ncn_plan_of_record.md

> **`NOTE:`** Several components below are necessary to provide redundancy in the event of hardware failure.

> Any of the disks may be used over the following busses:

Reviewer: My understanding is that both "busses" and "buses" are technically correct, but that "buses" is considered the more standard spelling.

```suggestion
Any of the disks may be used over the following buses:
```

- SAS
- SATA
- NVME

> > **`NOTE:`** USB is implicitly excluded during disk selection and wiping, the NCN's depoyment code will wipe all disks if they are a RAID or in the above list. The manual wipes will exclude USB, but it is recommended to verify manual wipes are actually doing so.

Reviewer: ```suggestion
> **NOTE:** USB is implicitly excluded during disk selection and wiping. The NCN's deployment code will wipe all disks if they are a RAID or in the above list.
> The manual wipes will exclude USB, but it is recommended to verify that the manual wipes are actually doing so.
```

> The OS disks are chosen by selecting the first two disks from a list containing all the disks in order by smallest to largest.

Reviewer: ```suggestion
The OS disks are chosen by selecting the smallest disks. Two disks are used for OS disks by default.
```
